cfme/common/provider_views.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
+from widgetastic.widget import View, Text
+
+from cfme import BaseLoggedInPage
+from widgetastic_patternfly import Dropdown, BootstrapSelect
+from widgetastic_manageiq import (BreadCrumb,
+                                  SummaryTable,
+                                  Button,
+                                  TimelinesView,
+                                  DetailsToolBarViewSelector,
+                                  ItemsToolBarViewSelector,
+                                  Checkbox,
+                                  Input,
+                                  Table,
+                                  PaginationPane,
+                                  FileInput,
+                                  Search,
+                                  DynaTree,
+                                  BootstrapTreeview)
+from widgetastic.utils import VersionPick, Version
+
+
+class ProviderDetailsToolBar(View):
+    """
+    represents provider toolbar and its controls
+    """
+    monitoring = Dropdown(text='Monitoring')
+    configuration = Dropdown(text='Configuration')
+    reload = Button(title='Reload Current Display')
```

Comment:
since class attributes do not shadow built-ins and reload no longer is a builtin in python3
i think that this one can stay